cob_script_server/src/simple_script_server/simple_script_server.py

```diff
@@ -1247,7 +1247,10 @@ def wait_for_finished(self, duration, logging):
 				if logging:
 					rospy.loginfo("Wait for <<%s>> reached <<%s>> (max %f secs)...",self.component_name, self.parameter_name,duration)
 				if not self.client.wait_for_result(rospy.Duration(duration)):
-					message = "Timeout while waiting for <<" + self.component_name +">> to reach <<" + self.parameter_name + ">>. Continuing..."
+					if type(self.parameter_name) is types.StringType:
+						message = "Timeout while waiting for <<" + self.component_name +">> to reach <<" + self.parameter_name + ">>. Continuing..."
+					else:
+						message = "Timeout while waiting for <<" + self.component_name +">>. Continuing..."
 					if logging:
```

Comment:
Yes, the @ipa-bnm solution is better , I actually did it checking the type of the parameter because so was already done on the  same script:

https://github.com/ipa320/cob_command_tools/blob/indigo_dev/cob_script_server/src/simple_script_server/simple_script_server.py#L1145

I will open a new pull request modifying both lines. Thanks @ipa-bnm